doc/taskwiki.txt

```diff
@@ -733,6 +745,13 @@ constructs.
         Example:
         let g:taskwiki_maplocalleader=",t"
 
+*taskwiki_taskopen_notes_folder*
+        The folder where taskopen has been configured to store notes. Defaults
+        to $HOME/tasknotes ().
+
+*taskwiki_taskopen_notes_regex*
+        The annotation to add for tasks that have a note. Defaults to Notes.
```

Comment:
Doesn't seem to be a `regex`, probably a leftover from early implementation? If I understand it correctly it's more like a marker for [default notes file](https://github.com/ValiValpas/taskopen#add-default-notes).

Reply:
This is after the `NOTES_REGEX` setting in taskopen (https://refined-github-html-preview.kidonng.workers.dev/ValiValpas/taskopen/raw/master/doc/html/taskopenrc(5).html).

In taskopen, it is meant as an actual regex, where you manually annotate a task and if the annotation matches the regex, then `taskopen -n` considers it as a task with a note.

The real problem is that taskopen does so much more that the use case covered here, which is limited to open an annotation with xdg-open and open/create a note with a specific annotation (no regex) in a standard format. 

It would be great if we could use taskopen to do these tasks instead of replicating a limited subset of its functionality. For the `TaskWikiOpen` command this is easy to do, just replace `xdg-open` with `taskopen`. But with the note, in order to have seemless integration, the note should be opened in a new buffer in the running vim/neovim instance. That, I had a hard time figuring out when it came to vim client/servers and neovim-remote and any assumptions about the user's setup, so I just implemented what seems a sufficient workaround. If you agree that this is sufficient, I can change the wording in the help file to be more clear.

Reply:
The new approach uses taskopen itself and this variable has been removed.